draft-ietf-rtcweb-jsep.xml

```diff
@@ -1064,10 +1072,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             immediately; if increased, additional candidates are pre-gathered;
             if decreased, the now-superfluous candidates are discarded.
             </t>
-            <t>Any changes to the BUNDLE policy take effect immediately, i.e.
+            <t>Any changes to the BUNDLE or RTCP multiplexing policies take effect immediately, i.e.
```

Comment:
FWIW I think frobbing these states post-construction is likely to lead to tears (both for implementors and app developers), so we should consider disallowing such changes altogether.